integration-tests/path-prefix/cypress/integration/path-prefix.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+
+describe(`Production pathPrefix`, () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it(`returns 200 on base route`, () => {
+    cy.url().should('include', '/blog')
+  })
+
+  describe('navigation', () => {
+    let anchor
+    beforeEach(() => {
+      anchor = cy.get('a[data-test-id="page-2-link"]')
+    })
+
+    it('prefixes link with /blog', () => {
+      anchor.should('have.attr', 'href').and('include', '/blog')
+    })
+
+    it('can navigate to secondary page', () => {
+      anchor.click()
+
+      cy.url().should('match', /\/blog\/page-2\/$/)
```

Comment:
There might be a better way to compare these here. Basically I want an "ends with" match.

Any folks with Cypress experience chiming in here would be appreciated :)

Reply:
we can use `cy.location(`pathname`).should(`equal`, `/blog/page-2`)`